### ISSUE TITLE
Update all samples to use client_secrets.json

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#156]
```
Update all samples to use client_secrets.json

Samples needing updating:
 audit
 moderator
 threadqueue
 urlshortener

```

Original issue reported on code.google.com by `jcgregorio@google.com` on 20 Jun 2012 at 2:05

**Comment by GoogleCodeExporter** [comment]
```
CL out for review: http://codereview.appspot.com/6305118/
```

Original comment by `jcgregorio@google.com` on 20 Jun 2012 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f62d0a489863.
```

Original comment by `jcgregorio@google.com` on 20 Jun 2012 at 4:11
- Changed state: **Fixed**
